src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseUserServiceImpl.java

```diff
@@ -137,7 +133,7 @@
 public abstract class BaseUserServiceImpl<U extends User, S extends Skill, SF extends SolrField> implements BaseUserService<U, S> {
     private static final Logger LOGGER = LoggerFactory.getLogger(BaseUserServiceImpl.class);
     private static final String USERS_DIR = "users";
-    private static final String AVATAR_EXT = "jpg";
+    private static final String AVATAR_CONTENT_TYPE = "image";
```

Comment:
this constant is already defined - com.odysseusinc.arachne.commons.utils.CommonFileUtils#TYPE_IMAGE just make it public